llvm/lib/ExecutionEngine/Orc/OrcV2CBindings.cpp

```diff
@@ -863,6 +866,31 @@ void LLVMOrcObjectLayerEmit(LLVMOrcObjectLayerRef ObjLayer,
       std::unique_ptr<MemoryBuffer>(unwrap(ObjBuffer)));
 }
 
+LLVMErrorRef
+LLVMOrcObjectLayerRegisterPluginJITLoaderGDB(LLVMOrcObjectLayerRef ObjLayer,
+                                             LLVMBool RequireDebugSectionsFlag,
+                                             LLVMBool AutoRegisterCodeFlag) {
```

Comment:
I guess these are the relevant flags for users. I didn't include the `RegistrationFunctionDylib` option for the executor side, because I don't have a use-case and I am not sure the required functionality for it is exposed either.